plugins/rplanners/rplanners.h

```diff
@@ -607,6 +608,10 @@ class SpatialTree : public SpatialTreeBase
         }
     }
 
+    ConstraintFilterReturnPtr GetConstraintReport() const {
```

Comment:
Return `const ConstraintFilterReturnPtr&`, you shouldn't need to modify the content of the report or the pointer.